### PR TITLE
[nrf toup] nrfx: update MDK to version 8.34.0

### PR DESCRIPTION
**Pull request by hubertmis**
MDK in version 8.34 adds EVENTS_SYNC for nRF52840. This event is needed
for 802.15.4 Wi-Fi Coexistence and Antenna Diversity features.

Signed-off-by: Hubert Miś <hubert.mis@nordicsemi.no>

### CODE REVIEW

File: nrfx/mdk/nrf5340_application_bitfields.h [hunk 1]
```diff
@@ -10732,13 +10707,6 @@ POSSIBILITY OF SUCH DAMAGE.
 #define QSPI_CINSTRDAT1_BYTE4_Pos (0UL) /*!< Position of BYTE4 field. */
 #define QSPI_CINSTRDAT1_BYTE4_Msk (0xFFUL << QSPI_CINSTRDAT1_BYTE4_Pos) /*!< Bit mask of BYTE4 field. */
 
-/* Register: QSPI_IFTIMING */
```
Comment (kl-cruz) [inline nrfx/mdk/nrf5340_application_bitfields.h:10735]: Please check compilation of this branch with QSPI example

File: nrfx/mdk/nrf5340_application_bitfields.h [hunk 2]
```diff
@@ -10547,26 +10523,25 @@ POSSIBILITY OF SUCH DAMAGE.
...
-/* Bits 31..28 : SCK frequency is given as 96 MHz / (SCKFREQ + 1). */
+/* Bits 31..28 : SCK frequency is derived from PCLK192M with SCK frequency = PCLK192M / (2*(SCKFREQ + 1)). */
 #define QSPI_IFCONFIG1_SCKFREQ_Pos (28UL) /*!< Position of SCKFREQ field. */
 #define QSPI_IFCONFIG1_SCKFREQ_Msk (0xFUL << QSPI_IFCONFIG1_SCKFREQ_Pos) /*!< Bit mask of SCKFREQ field. */
 
 /* Bit 25 : Select SPI mode. */
 #define QSPI_IFCONFIG1_SPIMODE_Pos (25UL) /*!< Position of SPIMODE field. */
 #define QSPI_IFCONFIG1_SPIMODE_Msk (0x1UL << QSPI_IFCONFIG1_SPIMODE_Pos) /*!< Bit mask of SPIMODE field. */
 #define QSPI_IFCONFIG1_SPIMODE_MODE0 (0UL) /*!< Mode 0: Data are captured on the clock rising edge and data is output on a falling edge. Base level of clock is 0 (CPOL=0, CPHA=0). */
-#define QSPI_IFCONFIG1_SPIMODE_MODE3 (1UL) /*!< Mode 3: Data are captured on the clock falling edge and data is output on a rising edge. Base level of clock is 1 (CPOL=1, CPHA=1). */
```
Comment (kl-cruz) [inline nrfx/mdk/nrf5340_application_bitfields.h:10569]: This is supported in QSPI driver and hal

**Comment by kl-cruz** [comment]
Commit message is incomplete. New MDK also comes with nRF5340's SYNC RADIO event support.

**Comment by hubertmis** [comment]
nRF5340 SYNC event was defined in previous MDKs. This MDK adds this event only to nRF52840

File: nrfx/mdk/nrf5340_network_bitfields.h [hunk 3]
```diff
@@ -8071,15 +7789,6 @@ POSSIBILITY OF SUCH DAMAGE.
...
 #define RESET_RESETREAS_RESETPIN_Detected (1UL) /*!< Detected */
 
-/* Register: RESET_NETWORK_FORCEOFF */
-/* Description: Force off power and clock in network core */
-
-/* Bit 0 : Force off power and clock in network core */
-#define RESET_NETWORK_FORCEOFF_FORCEOFF_Pos (0UL) /*!< Position of FORCEOFF field. */
-#define RESET_NETWORK_FORCEOFF_FORCEOFF_Msk (0x1UL << RESET_NETWORK_FORCEOFF_FORCEOFF_Pos) /*!< Bit mask of FORCEOFF field. */
-#define RESET_NETWORK_FORCEOFF_FORCEOFF_Release (0UL) /*!< Release force off signal */
-#define RESET_NETWORK_FORCEOFF_FORCEOFF_Hold (1UL) /*!< Hold force off signal */
```
Comment (anangl) [inline nrfx/mdk/nrf5340_network_bitfields.h:8081]: This needs a corresponding update (adding of `#if defined ...`) in `hal/nrf_reset.h`.

File: nrfx/mdk/nrf5340_application_bitfields.h [hunk 4]
```diff
@@ -10837,12 +10805,6 @@ POSSIBILITY OF SUCH DAMAGE.
 /* Register: RESET_RESETREAS */
 /* Description: Reset reason */
 
-/* Bit 27 : Reset from network CTRL-AP detected */
-#define RESET_RESETREAS_LCTRLAP_Pos (27UL) /*!< Position of LCTRLAP field. */
-#define RESET_RESETREAS_LCTRLAP_Msk (0x1UL << RESET_RESETREAS_LCTRLAP_Pos) /*!< Bit mask of LCTRLAP field. */
```
Comment (anangl) [inline nrfx/mdk/nrf5340_application_bitfields.h:10842]: Another reason to update `hal/nrf_reset.h`, and `helpers/nrfx_reset_reason.h` as well.

**Comment by hubertmis** [comment]
As this merge is not so trivial, decided to postpone it. We shall update MDK later, in a standard way.

**Pull request closed without merging**